cindex.py

```diff
@@ -143,6 +144,7 @@ def _start_indexing(self):
             cmd.append('-reset')
             cmd.extend(self._paths_to_index)
         proc = self._get_proc(cmd)
+        overall_start = time.time()
```

Comment:
How about using the timeit module and default_timer? Or since this is Python 3.3, time.perf_counter()?